sdk/typescript/src/providers/json-rpc-provider.ts

```diff
@@ -576,6 +579,25 @@ export class JsonRpcProvider extends Provider {
   }
 
   // Events
+  async getEvents(
+      query: EventQuery,
+      cursor: EventId| null,
+      limit: number|null,
+      descendingOrder: boolean| null
+  ): Promise<PaginatedEvents> {
+    try {
+      return await this.client.requestWithType(
+          'sui_getEvents',
+          [query, cursor, limit, descendingOrder],
```

Comment:
```suggestion
          [query, cursor, limit, descendingOrder],
```

Sorry just saw @stella3d's previous comment. If we are concerned about byte size for `descendingOrder`, we can replace `descendingOrder` with `order === 'descending'`  here.